test/Benchmarks/README.md

```diff
@@ -1,10 +1,7 @@
 # OpenTelemetry Benchmarks
 
-Use the following example to run Benchmarks from command line:
-(change parameters as necessary)
-
 Navigate to `./test/Benchmarks` directory and run the following command:
 
 ```sh
-dotnet run --framework net6.0 --configuration Release --filter *TraceBenchmarks*
+dotnet run -c Release -f net6.0 -- -m
```

Comment:
if we remove the filter , please add example with the filter as well below (to make it easy to copy paste to cmd).
Also - add a one line explaining why would folks want to use/not-use -m parameter.